apps/tutorial_servo_control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,31 @@
+#include "TutorialServo.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float minPulsewidthInMs , float maxPulsewidthInMs){
+        m_servoRangeInDegrees =  servoRangeInDegrees;
+        m_minPulsewidthInMs = minPulsewidthInMs;
+	m_maxPulsewidthInMs = maxPulsewidthInMs;
+}
+
+float TutorialServo::getServoRangeInDegrees() const{
+
+        return (m_servoRangeInDegrees);
+}
+
+float TutorialServo::getMinPulseWidthInMs()const{
+
+        return (m_minPulsewidthInMs);
+
+}
+float TutorialServo::getMaxPulseWidthInMs()const{
+
+        return (m_maxPulsewidthInMs);
+
+}
+void TutorialServo::setPositionInDegrees(float degrees){
+
+         float slope{0.0}, pulse{0.0}, range{m_maxPulsewidthInMs - m_minPulsewidthInMs};
```

Comment:
I don't think you need to initialize slope and pulse in this line and can just initialize them in the following lines. I also find the initialization you use interesting. Most of the code in the repo uses the lvalues = rvalues and its something to think about for consistency of code. I think clang-tidy doesn't like initializing multiple variables on a single line so you may run into compiler warnings about that.